src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -496,22 +494,6 @@ private static OutputStream getOutputStream(Path outputPath) throws IOException
         return result;
     }
 
-    /**
-     * Create {@link AutomaticBean.OutputStreamOptions} for the given location.
-     * @param outputPath output location
-     * @return output stream options
-     */
-    private static AutomaticBean.OutputStreamOptions getOutputStreamOptions(Path outputPath) {
```

Comment:
Is there any history to this code on why it was added since you are removing it and some tests.

Reply:
long story of refactoring ... , initially was added here https://github.com/checkstyle/checkstyle/commit/1cea3765893e83e0d58ca110113e0d05b15a798a#diff-1bccd470310a92a42dcea475324b54adR114 , probably at that time (17 years ago), there was some problem with closing `System.out`, such code was added as part of big update.

I do not think such is required.

Reply:
Closing the output stream (even the `System.out`) will discard all future output. It's not a problem anymore. Because all additional output has been sent to the `system.err`. But this is a questionable practice. The stream must be closed by the one who opened it. There may arise problems in the future. Such as partially lost output.

Reply:
just tested it and it is actually discarded (no exceptions, just discarded). We are ok now only because we moved extra output to `err`, but reverting of ebe80e505623cebde3799a72ede68dcc964ef10f make problem visible.
So to kill such mutation we would need to make reflection based UT, just to make pitest happy.